sycamore/transforms/embed.py

```diff
@@ -56,7 +56,7 @@ def generate_embeddings(self, doc_batch: list[Document]) -> list[Document]:
 
         assert self._transformer is not None
 
-        text_batch = [doc.text_representation for doc in doc_batch]
+        text_batch = ["" if doc.text_representation is None else doc.text_representation for doc in doc_batch]
```

Comment:
What's the right thing to do here longterm? In this case we will get an embedding for the empty string ingested into OpenSearch, right? I wonder if eventually we will need a way to skip elements that aren't text. Not sure if that's right either, just thinking out loud...

Reply:
yes, we should not embed in this case, in long term, I assume we should check based on the document type by each different embedder.